service/src/main/resources/changelog/changesets/20220823_run.yaml

```diff
@@ -0,0 +1,41 @@
+databaseChangeLog:
+  - changeSet:
+      id: "1"
+      author: chrisl
+      changes:
+        - createTable:
+            tableName: run
+            columns:
+              - column:
+                  name: id
+                  type: uuid
+                  constraints:
+                    primaryKey: true
+                    nullable: false
+              - column:
+                  name: engine_id
+                  type: varchar(255)
+                  constraints:
+                    nullable: true
```

Comment:
Won't this also be a UUID and not nullable?

Reply:
In Cromwell, yes it's a UUID... in other engines I'm not sure.

In the future we may end up generating workflows in CBAS before we submit them to the engine (ie the submit queue). If WES can be modified to have a "submit your own ID" option then we could pregenerate our own UUIDs, but until then we might need to be able to create workflows before we get workflow IDs back from the engine